### PR TITLE
Add limit to clickhouse sql events

### PR DESCRIPTION
**Pull request by liyiy**
## Changes

For the definitions drawer, we include a snippet of the events table. We want to be able to add a limit to the Clickhouse api call for events so we don't return unnecessary data

Screen Shot 2021-06-14 at 10 58 47 AM

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious
- [ ] Frontend/CSS is usable at 320px (iPhone SE) and decent at 360px (most phones)

### CODE REVIEW

**Comment by liyiy** [comment]
I don't have clickhouse on my machine, will need someone to look this over

File: ee/clickhouse/views/events.py [hunk 1]
```diff
@@ -46,7 +46,8 @@ def _get_people(self, query_result: List[Dict], team: Team) -> Dict[str, Any]:
     def _query_events_list(
         self, filter: Filter, team: Team, request: Request, long_date_from: bool = False, limit: int = 100
     ) -> List:
-        limit_sql = f"LIMIT {limit + 1}"
+        limit += 1
+        limit_sql = f"LIMIT %(limit)"
```
Comment (EDsCODE) [inline ee/clickhouse/views/events.py:50]: no need for f here! The parameter is specified by %()s so it should be `%(limit)s`

**Review by EDsCODE — CHANGES_REQUESTED** [review]

File: posthog/api/test/test_event.py [hunk 2]
```diff
@@ -422,6 +422,34 @@ def test_get_event_by_id(self):
             response = self.client.get(f"/api/event/im_a_string_not_an_integer",)
             self.assertIn(response.status_code, [status.HTTP_404_NOT_FOUND, status.HTTP_400_BAD_REQUEST])
 
+        def test_limit(self):
```
Comment (buwilliams) [inline posthog/api/test/test_event.py:425]: Added a test to ensure it works!

**Review by buwilliams — APPROVED** [review]
Once tests pass I think we are good to go.

File: posthog/api/test/test_event.py (hunk 2 above)
Comment (liyiy) [reply posthog/api/test/test_event.py:425]: thank you good sir

**Pull request merged**